modules/users/client/controllers/admin/view-applicants.client.controller.js

```diff
@@ -30,27 +47,15 @@
       if (user) {
           var newUser = user;
           user.approvedStatus = true;
+          user.roles = user.roles.id;
```

Comment:
the user.roles attribute is an array correct? If so, then this will change it to a string value

Reply:
user.roles is a json object with both an id and a name. The name is outputted on the view and the id is the role we end up using on the database, so this is the line where we drop the frontend-friendly part of the object.

There should be a comment here, I'll add one.